packages/components/checkbox/index.js

```diff
@@ -71,47 +77,49 @@ export const Checkbox = ({
       ]),
     },
     label: {
-      ml: isReversed ? '0' : '2',
-      mr: isReversed ? '2' : '0',
-      fontFamily: 'body',
-      fontSize: 'checkbox',
-      fontWeight: 'medium',
-      letterSpacing: 'checkbox',
-      color: `${
-        isDisabled ? 'font.checkbox.disabled' : 'font.checkbox.default'
-      }`,
+      ml: isReversed ? '0' : label.normal.margin,
```

Comment:
0 is not from space in theme ?

Reply:
The control on that is not needed IMO, this can be hardcoded in our component.